ui-packages/packages/trusty/src/components/Organisms/ExecutionHeader/ExecutionHeader.tsx

```diff
@@ -30,7 +31,11 @@ const ExecutionHeader = (props: ExecutionHeaderProps) => {
             />
           )}
           {execution.status === RemoteDataStatus.SUCCESS && (
-            <Title size="3xl" headingLevel="h2">
+            <Title
+              size="3xl"
+              headingLevel="h2"
+              {...attributeOuiaId('execution-title')}
```

Comment:
I think you can use function `componentOuiaProps` instead (to set both ouiaType and ouiaId). But use it in the top-level component being rendered - which here is the wrapping `<section className="execution-header">`. Please keep ouiaType value in sync with component name, perhaps `'execution-header'` in this case.
Cause semantically it probably does not make sense to have an element with ouiaId, but without ouiaType.

Reply:
And moreover Title is already an OUIA compliant component - instead of your current configuration you can use:
```suggestion
              ouiaId="execution-title"
```
If that's what you want.